content/docs/user-guide/packaging/project-export/project-export-android.md

```diff
@@ -0,0 +1,117 @@
+```
+"assets": "android",
+```
+6. Edit the `$O3DE_ENGINE_PATH/Registry/AssetProcessorPlatformConfig.setreg` file at line 67 by uncommenting the line:
+```
+//"android": "enabled", -> "android": "enabled",
+```
+This will tell the asset processor to make sure ios assets are cached when building the project.
+
+7. Make sure your project has the right configuration settings for Android. You can use the `android-configure` CLI tool to ensure this is setup. To see what is currently configured, run:
+```
+$O3DE_ENGINE_PATH/scripts/o3de.bat android-configure -l
+```
+You can also validate your current configuration using:
+```
+$O3DE_ENGINE_PATH/scripts/o3de.bat android-configure --validate
+```
+If you do not already have android configured, or encounter issues with the validate command, you can run the following:
+```
+$O3DE_ENGINE_PATH/scripts/o3de.bat android-configure --set-value platform.sdk.api=30
+$O3DE_ENGINE_PATH/scripts/o3de.bat android-configure --set-value ndk.version=25.*
+$O3DE_ENGINE_PATH/scripts/o3de.bat android-configure --set-value android.gradle.plugin=8.1.0
+$O3DE_ENGINE_PATH/scripts/o3de.bat android-configure --set-value sdk.root=/path/to/android/sdk
+$O3DE_ENGINE_PATH/scripts/o3de.bat android-configure --set-value signconfig.store.file=/path/to/android-key.keystore
+$O3DE_ENGINE_PATH/scripts/o3de.bat android-configure --set-value signconfig.key.alias=o3dekey
+$O3DE_ENGINE_PATH/scripts/o3de.bat android-configure --set-value asset.mode=PAK
+```
+Note that for sdk.root and signconfig.store.file, you must already know where those things exist on your hard drive, and use those paths accordingly. To setup the keypass and storepass, you must use the following commands
+```
+$O3DE_ENGINE_PATH/scripts/o3de.bat android-configure --set-password signconfig.store.password
+$O3DE_ENGINE_PATH/scripts/o3de.bat android-configure --set-password signconfig.key.password
+```
+Make sure the passwords supplied here match what you used in the keystore file, and that you remember what they are.
+
+## Quickstart
+### Running the Export Script
+For building in release mode with bundled PAK files for your APK, you can use the following export command:
+```
+export O3DE_ENGINE_PATH=/path/to/o3de
+export PROJECT_PATH=/path/to/project
+cd $PROJECT_PATH
+$O3DE_ENGINE_PATH/scripts/o3de.sh export-project -es export_source_android.py -pp . -abp build/game-android -ll INFO --config release --asset-mode PAK
+```
+
+To use profile mode instead, change the `--config` parameter in the above snippet to use `profile` instead. If you wish to set the asset mode to LOOSE, use `--asset-mode LOOSE`.
+
+That invocation should be all that is needed to create an APK file for your project. To install the APK on your phone for testing, this can be done by adding the `--deploy-to-android` flag in the export snippet above. Please note that to use this feature, you must ensure an android device is connected to your computer, and that ADB is able to properly access the device.
```

Comment:
_"That invocation should be all that is needed to create an APK file for your project"_
😄 👍